SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -786,11 +791,19 @@ int32 USpatialNetDriver::ServerReplicateActors_ProcessPrioritizedActors(UNetConn
 				}
 			}
 
+			// SpatialGDK - Creation of new entities should always be handled and therefore is checked prior to actor throttling.
+			// There is a EntityCreationRateLimit to prevent overloading spatial with creation requests if the developer desires.
+			// Creation of a new entity occurs when the channel is currently nullptr or if the channel does not have bCreatedEntity set to true.
+			if (FinalCreationCount < MaxEntitiesToCreate && !Actor->GetTearOff() && (Channel == nullptr || (Channel != nullptr && !Channel->bCreatedEntity)))
```

Comment:
`bCreatedEntity` is set to true on create entity response, we will be counting them every frame until they receive the response. Use `bCreatingNewEntity` instead (set to true in `SetChannelActor`, set to false right after sending create entity request)
```suggestion
			if (FinalCreationCount < MaxEntitiesToCreate && !Actor->GetTearOff() && (Channel == nullptr || Channel->bCreatingNewEntity))
```